madeofhuman/maintenance-react
# #161042121 fix breaking pagination

#### What does this PR do?
- fix breaking pagination and 

#### Description of Task to be completed?
- The request table fills up the display and does not scroll down or paginate

#### How should this be manually tested?
- visit https://maintenance-react-staging.herokuapp.com and create > 20 requests to see the pagination at work

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161042121](https://www.pivotaltracker.com/story/show/161042121)

#### Screenshots
![screen shot 2018-10-08 at 2 58 51 am](https://user-images.githubusercontent.com/7345686/46589823-25a8c700-caa6-11e8-8d5c-591543d9450d.png)

#### Questions:
N/A

[Delivers #161042121]
